code/modules/mob/living/carbon/carbon.dm

```diff
@@ -192,6 +192,8 @@
 			power_throw++
 		if(neckgrab_throw)
 			power_throw++
+		do_attack_animation(target, no_effect = 1) //PARIAH EDIT ADDITION - AESTHETICS
+		playsound(loc, 'sound/weapons/punchmiss.ogg', 50, TRUE, -1) //PARIAH EDIT ADDITION - AESTHETICS
```

Comment:
Aesthetics?

Reply:
Some idea how to better call this, perhaps?

Reply:
I'm not sure if it needs a label at all really. AFAIK they're usually only used to point to the folder of the module that added the change, whereas this is something being added separately.
It doesn't make any difference either way though, and I could be wrong with the label thing.